parent/pom.xml

```diff
@@ -2484,6 +2498,7 @@
                   <dep>org.apache.logging.log4j:log4j-slf4j2-impl</dep>
                   <dep>org.apache.logging.log4j:log4j-core</dep>
                   <dep>io.camunda:zeebe-build-tools</dep>
+                  <dep>io.camunda:zeebe-cluster-config</dep>
```

Comment:
ℹ️ The dependency check of the `gateway-grpc` module otherwise fails since that module doesn't use any of those classes directly, only in the inheritance chain of certain classes. Removing the dependency from the module would fail its compilation.